docs/UserGuide.md

```diff
@@ -43,12 +43,15 @@ If you encounter any issues in launching and using the app, feel free refer to t
 * Items with `...` after them can be used multiple times.
   * e.g. `[c/CLASS_NAME...]` can be used as ` ` (i.e. 0 times since it is also optional), `c/CS1101S`, `c/CS2030 c/ST2334` etc.
 * Parameters, excluding index parameters, can be in any order.
-  * e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  * e.g. if you specify `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```

Comment:
If we are changing the instruction to the reader's perspective, the word 'specify' here might sound weird. Also, in this case it is more like comparing what the command expects and how the user can enter it in another format, so I believe retaining `the command specifies` might be better.

Reply:
Fair point.